src/style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+}
+
+.search_bar {
+  width: 100%;
+  position: relative;
+}
+
+.search_barBig_input {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.image_findBig {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search_barSmall_input {
+  position: relative;
```

Comment:
try to set the position of search img (absolute) with the same logic in both inputs (i.e position `relative` in the according elements)

Reply:
thank you for the correction
I will keep it in mind